### DISCUSSION TITLE
Feature Request fn

### DISCUSSION THREAD

**Discussion by Dairoe** [vlang/v#18641] [Ideas]
### Describe the bug

error is not clear enough, 

i know you can use trail syntax , to do same exact thing but what if want different way of doing it.

https://play.vlang.io/p/3ce0e46f0f

chose wrong label someone update this to feature request. thanks

### Expected Behavior

```
main.v:18:7: error: cannot use field annotation on fucntion call
   16 | 
   17 | fn main() {
   18 |     a := new_player(name: 'test', health: 100)
      |          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   19 | }
```

### Current Behavior

```
main.v:18:7: error: expected 2 arguments, but got 1
   16 | 
   17 | fn main() {
   18 |     a := new_player(name: 'test', health: 100)
      |          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   19 | }
```

### Reproduction Steps

above link

### Possible Solution

i think it should be allowed because it can also help ide's to perform auto completion like it does with structs.

_No response_

### V version

0.3.2

### Environment details (OS name and version, etc.)

Linux

**Comment by changrui** [comment]
`a := new_player('test', 100)` is running well.

**Comment by MatejMagat305** [comment]
> `a := new_player('test', 100)` is running well.

but it means the error messenge(report) is unhelpfull and I agree ....
